amitools/vamos/lib/DosLibrary.py

```diff
@@ -30,8 +30,8 @@ class DosLibrary(AmigaLibrary):
 
   DOSFALSE = 0
   DOSTRUE = 0xffffffff
-  
-  LV_VAR   = 0	# an variable 
+
+  LV_VAR   = 0	# an variable
```

Comment:
All changes in this file were performed automatically by EMACS and are purely cosmetic.